scripts/update/Updater.php

```diff
@@ -843,6 +843,6 @@ public function update($initialVersion) {
             $this->setVersion('5.38.2');
         }
 
-        $this->skip('5.38.2', '5.38.4');
+        $this->skip('5.38.2', '5.39.0');
```

Comment:
You should also provide an update script to ensure the default values are there.